### DISCUSSION TITLE
Testers required

### DISCUSSION THREAD

**Discussion by matt2005** [opencardev/crankshaft#531] [Ideas]
Hi,

I'm working on a new release which will bring back Pi2 support and improve AASDK.
It should be ready for testing in the next couple of weeks.

I would like to release it to a small set of users before I fully release it.

Please note it may not work and is a Work in progress (WIP)

**Comment by mateuszjankaminski** [comment]
how can I hel you?

**Comment by F4u5t** [comment]
i will help

**Comment by AnastasiaDen1** [comment]
У нас сейчас в работе вакансии QA C# (Мидл и Лид) в Альфа банк

💡Вакансия: QA Engineer C# 
 🖥 Формат: полная удаленка
 📍 Проект: Кредитование юридических лиц (ККБ, СКБ) 
   
📌Обязанности: 
Функциональное тестирование 
Написание тестовой документации (сценарии, отчеты по тестированию); 
Поддержка и развитие тестового фреймворка на C#, .net; 
Соотношение 50% ручное тестирование, 50% автоматизированное тестирование 

📌 Требования: 
Опыт ручного тестирования 1-2 года 
Опыт автоматизации тестирования на C# от 6 месяцев 
Опыт тестирования API 
Практический опыт применения практик тест-дизайна 
Рабочие знания клиент-серверной архитектуры 
Умение проектировать тест-кейсы и готовить тестовые данные для тестирования 
Опыт работы с SOAP, REST, SQL 

⚡️Условия: 
Полная удаленка; 
Заботу о твоем здоровье: ДМС, оплата больничного, скидки на фитнес; 
Участие в митапах и программах обучения Альфа-Банка; 
Возможность как вертикального, так и горизонтального карьерного роста

Связаться по вакансии можно через электронную почту: Denisova@ascr.ru 
или
TG: @DjunTao
